cluster-state-service/handler/reconcile/loader/taskloader.go

```diff
@@ -148,6 +153,31 @@ func (loader taskLoader) getTasksFromECS(cluster *string) ([]types.Task, error)
 	return tasks, nil
 }
 
+// getTaskARNsFromECS gets a list of the task ARNs from ECS for both running and stopped tasks.
+// ECS currently stores stopped tasks for about 1 hour.
```

Comment:
The API documentation at http://docs.aws.amazon.com/cli/latest/reference/ecs/list-tasks.html says `Recently-stopped tasks might appear in the returned results. Currently, stopped tasks appear in the returned results for at least one hour`. But I agree, since it is non-deterministic, I have removed the comment.